.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+README.md
+            - ชื่อ: https://github.com
+  การใช้งาน: actions/upload-artifact@v4.3.6
+  กับ:
+    # ชื่อสิ่งประดิษฐ์
+    ชื่อ: # ตัวเลือก ค่าเริ่มต้นคือสิ่งประดิษฐ์
+    # ไฟล์ ไดเรกทอรี หรือรูปแบบไวด์การ์ดที่อธิบายถึงสิ่งที่จะอัปโหลด
+    เส้นทาง:
+    # พฤติกรรมที่ต้องการหากไม่พบไฟล์จากเส้นทางที่ให้มา
+ตัวเลือกที่มี:
+  warn: แสดงคำเตือนแต่ไม่ล้มเหลวในการดำเนินการ
+  ข้อผิดพลาด: การดำเนินการล้มเหลวโดยมีข้อความแสดงข้อผิดพลาด
+  ไม่สนใจ: ไม่แสดงคำเตือนหรือข้อผิดพลาดใด ๆ การกระทำจะไม่ล้มเหลว
+
+    if-no-files-found: # ตัวเลือก ค่าเริ่มต้นคือ warn
+    # ระยะเวลาที่สิ่งประดิษฐ์จะหมดอายุเป็นวัน 0 หมายถึงใช้การคงไว้เป็นค่าเริ่มต้น
+ขั้นต่ำ 1 วัน สูงสุด 90 วัน เว้นแต่จะเปลี่ยนแปลงจากหน้าการตั้งค่าที่เก็บข้อมูล
+
+    วันเก็บรักษา: # ตัวเลือก
+    # ระดับการบีบอัดสำหรับ Zlib ที่จะนำไปใช้กับไฟล์เก็บถาวรอาร์ทิแฟกต์ ค่าสามารถอยู่ระหว่าง 0 ถึง 9: - 0: ไม่มีการบีบอัด - 1: ความเร็วที่ดีที่สุด - 6: การบีบอัดเริ่มต้น (เช่นเดียวกับ GNU Gzip) - 9: การบีบอัดที่ดีที่สุด ระดับที่สูงกว่าจะส่งผลให้การบีบอัดดีขึ้น แต่จะใช้เวลาในการดำเนินการนานกว่า สำหรับไฟล์ขนาดใหญ่ที่ไม่สามารถบีบอัดได้ง่าย แนะนำให้ใช้ค่า 0 เพื่อการอัปโหลดที่เร็วขึ้นอย่างเห็นได้ชัด
+
+    ระดับการบีบอัด: # ตัวเลือก ค่าเริ่มต้นคือ 6
+    # หากเป็นจริง สิ่งประดิษฐ์ที่มีชื่อตรงกันจะถูกลบออกก่อนที่จะอัปโหลดสิ่งประดิษฐ์ใหม่ หากเป็นเท็จ การดำเนินการจะล้มเหลวหากมีสิ่งประดิษฐ์สำหรับชื่อที่กำหนดไว้แล้ว จะไม่ล้มเหลวหากไม่มีสิ่งประดิษฐ์ดังกล่าว
+
+    เขียนทับ: # ตัวเลือก ค่าเริ่มต้นคือเท็จ
```

Comment:
To [Goutam Tamvada](https://github.com/xvzcf) and [Douglas Stebila](https://www.douglas.stebila.ca/) to develop VeriFrodo, an open-source package implementing a lattice-based quantum-resistant cryptographic algorithms in Jasmin within the [Open Quantum Safe](https://github.com/open-quantum-safe) project.